cumulusci/tasks/metadata_etl/base.py

```diff
@@ -305,3 +306,97 @@ def _transform(self):
                 removed_api_names.add(api_name)
 
         self.api_names = self.api_names - removed_api_names
+
+
+class UpdateMetadataFirstChildTextTask(MetadataSingleEntityTransformTask):
+    task_docs = """
+MetadataSingleEntityTransformTask to update a single child's text.  Specify:
+
+- ``metadata_type`` as the Metadata's `Metadata Type <https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_types_list.htm>`_
+
+- ``tag`` as the targeted child's tag
+
+- ``value`` as the new value of the child's text
+
```

Comment:
```suggestion
Metadata ETL task to update a single child element's text within metadata XML.
```

Task docs should not assume the reader is a Python developer familiar with CumulusCI internals.

And the task options are automatically listed below by the doc generator (based on `task_options`), so there's no need to repeat them.